### PR TITLE
Update windows.md

### PR DESCRIPTION
**Pull request by mozok**
Add example of using ImageMagick suit to convert PNG to ICO.

### CODE REVIEW

File: docs/en/manuals/windows.md [hunk 1]
```diff
@@ -17,5 +17,17 @@ The application icon used for a Windows game must be in the .ico format. You can
 
 Source: [Microsoft - Windows app icon construction](https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-construction#icon-sizes-win32)
 
+### Creating .ico file locally using [ImageMagick](https://www.imagemagick.org/) software suite.
+Example for Linux:
+1. Instal ImageMagick:
+```
+sudo apt install imagemagick
+```
+3. Prepare your PNG icon.
```
Comment (britzl) [inline docs/en/manuals/windows.md:26]: Where is 2. ?

File: docs/en/manuals/windows.md [hunk 2]
```diff
@@ -17,5 +17,17 @@ The application icon used for a Windows game must be in the .ico format. You can
 
 Source: [Microsoft - Windows app icon construction](https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-construction#icon-sizes-win32)
 
+### Creating .ico file locally using [ImageMagick](https://www.imagemagick.org/) software suite.
+Example for Linux:
+1. Instal ImageMagick:
```
Comment (britzl) [inline docs/en/manuals/windows.md:22]: Instal -> Install

Comment (britzl) [reply docs/en/manuals/windows.md:22]: Also, please move the ImageMagick link from the title to here

**Review by britzl — CHANGES_REQUESTED** [review]
See comments

**Comment by mozok** [comment]
Fixed.
*Note to myself: don't do PRs on the Friday evening after release week.

**Review by britzl — APPROVED** [review]
Thanks!

**Pull request merged**